core/src/scene/sceneLoader.cpp

```diff
@@ -101,7 +101,15 @@ bool SceneLoader::loadScene(Node& config, Scene& _scene) {
     std::sort(_scene.styles().begin(), _scene.styles().end(),
               [](std::unique_ptr<Style>& a, std::unique_ptr<Style>& b) {
                   if (a->blendMode() != b->blendMode()) {
-                      return a->blendMode() == Blending::none;
+                      if (a->blendMode() == Blending::none || b->blendMode() == Blending::none) {
+                          return a->blendMode() == Blending::none;
```

Comment:
Tangram-JS compares the style names as a tie-breaker [here](https://github.com/tangrams/tangram/pull/223/files#diff-e90eeb4f0c5a50cd03b669245e7e0724R428); we may as well do the same.